src/mod_carboncopy.erl

```diff
@@ -25,16 +25,16 @@
 %%%----------------------------------------------------------------------
 -module (mod_carboncopy).
 -author ('ecestari@process-one.net').
--xep([{xep, 280}, {version, "0.10"}]).
+-xep([{xep, 280}, {version, "0.6"}]).
+-xep([{xep, 280}, {version, "0.13.3"}]).
```

Comment:
Two different versions for the same xep?

Reply:
Yes, because 0.6 had different rules on what should be forwarded as a carbon.

Reply:
Discussed the NS issue, after the fix it looks ok.